js/main.js

```diff
@@ -0,0 +1,42 @@
+const inputTitle = document.getElementById('title');
+const inputAuthor = document.getElementById('author');
+const inserBooktBtn = document.getElementById('insertBtn');
+const listOutput = document.getElementById('lsOutput');
+
+const booklist = JSON.parse(localStorage.getItem('Books')) || [];
+
+inserBooktBtn.onclick = () => {
+  const title = inputTitle.value;
+  const author = inputAuthor.value;
+  if (title === '' || author === '') {
+    alert('Inputs cannot be empty');
+  } else {
+    const newBook = {
+      title,
+      author,
+    };
+    booklist.unshift(newBook);
+    localStorage.setItem('Books', JSON.stringify(booklist));
+    // eslint-disable-next-line no-restricted-globals
+    location.reload();
+  }
+};
+
+// eslint-disable-next-line no-unused-vars
+function removeBook(val) {
+  const book = booklist.filter((data, i) => i !== val);
+  localStorage.setItem('Books', JSON.stringify(book));
+  // eslint-disable-next-line no-restricted-globals
+  location.reload();
```

Comment:
- [ ] Linters help us write clean code. Let's maintain that. Kindly remove ESLint disabling here. For the resulting eslint error you can use **`window.location.reload()`** instead.